ci/kuiper/build_scopy_kuiper.sh

```diff
@@ -8,17 +8,17 @@ JOBS=-j1
 
 USE_STAGING=OFF
 
-LIBIIO_VERSION=master
+LIBIIO_VERSION=v0.25
 LIBM2K_BRANCH=master
 SPDLOG_BRANCH=v1.x
 VOLK_BRANCH=main
 GNURADIO_BRANCH=maint-3.10
 GRSCOPY_BRANCH=3.10
 GRM2K_BRANCH=master
 LIBSIGROKDECODE_BRANCH=master
-QWT_BRANCH=qwt-multiaxes
+QWT_BRANCH=qwt-multiaxes-updated
 LIBTINYIIOD_BRANCH=master
-
+IIOEMU_BRANCH=master
```

Comment:
could we change the reference to "main"? (i updated the default branch on iio-emu to be main now)
we should do this for all that apply